code/iaas/storage-service/pom.xml

```diff
@@ -31,7 +31,7 @@
         <dependency>
         	<groupId>io.cattle</groupId>
         	<artifactId>cattle-docker-common</artifactId>
-        	<version>0.5.0-SNAPSHOT</version>
+            <version>${project.version}</version>
```

Comment:
Unrelated to the bug, but stumbled upon this misuse of version and wanted to fix it.